tests/unit_tests/test_wave_02.py

```diff
@@ -34,7 +34,12 @@ def test_get_no_matching_items_by_category():
 
     items = vendor.get_by_category("electronics")
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
+    assert len(items) == 0
+    assert item_a not in items
+    assert item_c not in items
+    assert item_b not in items
```

Comment:
This works, but we can make it simpler by removing lines 38-40. If the length of `items` is 0, we know that none of our items can be present in it.